src/seedu/addressbook/AddressBook.java

```diff
@@ -1150,8 +1151,9 @@ private static String getUsageInfoForExitCommand() {
      * @param sign  Parameter sign to be removed
```

Comment:
method comments must be also updated if there are changes to behaviour or method signature